### DISCUSSION TITLE
Filtering, grouping, and cross referencing plans

### DISCUSSION THREAD

**Discussion by awm33** [CMSgov/price-transparency-guide#681] [General]
Unless the HIOS is used, it's difficult to tell which plans are employer health plans and which are health plans that are using an EIN instead of HIOS. We've been able to use the EINs for public companies and non-profit, but EINs for private companies are secret. That leaves us fuzzy matching names.

Has anyone else been able to identify / filter out private employer plans?

**Comment by bryanbriegel** [comment]
We have advocated for CMS to provide clarification in the schema/layout to ensure ready attribution of the rates to payers.  plan_name is  "The plan name and name of plan sponsor and/or insurance company" - where Plan ID Type is EIN (Employer) we would expect the plan name field to include the name of the plan sponsor.  We believe the OR condition (or insurance company) refers to the case when this is an issuer's set of rates and when the Plan ID Type is HIOS. This doesnt solve what you're looking for, however there is a need to ensure the files are populated/labeled for attribution by the public, which is the Department's intent, from our read of the rule and ICR.
